en/mulesoft/modules/ROOT/pages/ms_composer_custom_expression_editor.adoc

```diff
@@ -96,6 +96,15 @@ locale: String |DateTime
 `fromStringToNumber(0.05) = 0.05` |
 NA |text: String |Number
 
+|emptyString | Returns an empty string. +
+
+The emptyString function can be used wherever an empty string is needed, including as an input parameter in other functions. |`-` |`-` | String|
```

Comment:
```suggestion
You can use the emptyString function wherever an empty string is needed, including as an input parameter in other functions. |`-` |`-` | String|
```

Reply:
Or maybe "The emptyString function works wherever an empty string is needed, including as an input parameter in other functions."